cmd/schema.go

```diff
@@ -46,38 +46,66 @@ environment variables. The schema flags are:
 
 // SetFlags sets the flags.
 func (cmd *SchemaCmd) SetFlags(f *flag.FlagSet) {
+	f.StringVar(&cmd.source, "source", "", "Flag for specifying source DB, (accepted values are \"PostgreSQL\", \"MySQL\", and \"DynamoDB\")")
```

Comment:
Given that we accept other values than just these three, maybe let's be less prescriptive here and instead of "accepted values" just say:
e.g.  \"PostgreSQL\", \"MySQL\", and \"DynamoDB\"
Also, how about using ` strings here to make this string a bit easier to read.

Reply:
Done.